netuitive/conf/collectors/HttpdCollector.conf

```diff
@@ -1,2 +1,2 @@
 enabled = False
-urls = localhost http://localhost:8080/server-status?auto
+urls = localhost http://localhost:8080/server-status?auto,
```

Comment:
Why a comma at the end and not after `localhost`?

Reply:
According to the Diamond docs, the collector expects an entry to be formatted:     'nickname http://host:port/server-status?auto,'

http://diamond.readthedocs.io/en/latest/collectors/HttpdCollector/